README.md

```diff
@@ -317,7 +320,12 @@ $ curl -sk https://<bootstrap-node-url>/chainweb/0.0/mainnet01/cut | jq '.height
 
 ## Mine for a Chainweb Network
 
```

Comment:
```suggestion

Successful mining on mainnet requires specialized hardware (ASIC). The setup for solo mining involves a running chainweb-node with a configuration that enables mining and a [chainweb-mining-client](https://github.com/kadena-io/chainweb-mining-client/), which connects to the mining API of chainweb-node and provides a Stratum API for the mining hardware (ASIC).  
```

Reply:
Added in the latest commit with some small grammar changes